### ISSUE TITLE
injected License object attribute unitOfMeasure not valid when liceseType is regKey

### ISSUE THREAD

**Issue by jgruber** [F5Networks/f5-declarative-onboarding#26]
submitted declaration License object:

```
        "licenseKey": {
            "licenseType": "regKey",
            "class": "License",
            "regKey": "BUKNP-EAXJD-AKKVF-IDSYT-FAYLEUR",
            "overwrite": false
        },
```
produces an output declaration object with License object:

```
            "licenseKey": {
                "licenseType": "regKey",
                "class": "License",
                "regKey": "BUKNP-EAXJD-AKKVF-IDSYT-FAYLEUR",
                "overwrite": false,
                "unitOfMeasure": "monthly"
            },
```
 which when validated against:

```
https://raw.githubusercontent.com/F5Networks/f5-declarative-onboarding/master/schema/base.schema.json
```

Generates a validation error: 'Property unitOfMeasure is not allowed'

All injected default values MUST conform to schema validation.

**Comment by focrensh** [comment]
AT-212 for internal tracking

**Comment by seattlevine** [comment]
Resolved in 1.5.0
